### PR TITLE
Player roles, fixes #35

### LINKED ISSUE #35: Ämter / Beauftragte
- Mitglieder halten 1-n Rollen im Verein inne
- Soll bei Mitglieder Anzeige angezeigt werden
- Mit History (Rolle von Datum bis Datum)

### PR DESCRIPTION
**Pull request by DenniJensen**
This MR contains certain changes

1. it allows the admin to create roles for the club
2. the admin is able to allocate the roles for a user
3. Admin has now a `Admin Dashboard`
3. the attributes contained by the player are separated into:
  - contact
  - member
  - address
  - player
4. development seeding. This allows to recreate the database for new migrations and rollback testing

~~The migration were not tested with the production database. The translations are missing in the assignment view for roles.~~

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -62,6 +62,15 @@
 
   resources :teams, except: :show
 
+  resources :administrations, only: :index
```
Comment (elgrillo83) [inline config/routes.rb]: better use a singular resource here, see http://guides.rubyonrails.org/routing.html#singular-resources
`resource :administration, only: :show`
* the route will have a better wording: `/administration` 
* controller name, view paths, etc. stay the same

File: db/migrate/20160815160730_create_functions.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateFunctions < ActiveRecord::Migration[5.0]
+  def change
+    create_table :functions do |t|
+      t.belongs_to :user, foreign_key: true
+      t.belongs_to :role, foreign_key: true
+      t.date :assumed_at
+      t.date :vacated_at
```
Comment (elgrillo83) [inline db/migrate/20160815160730_create_functions.rb]: I recently read in a styleguide:
* use suffix `_at` for `datetime`
* use suffix `_on` for `date`
* use suffix `_time` for `time`
Which makes a lot of sense to me :-)

File: app/models/function.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+class Function < ApplicationRecord
+  belongs_to :user
+  belongs_to :role
+
+  before_save :correct_time_order?
```
Comment (DenniJensen) [inline app/models/function.rb]: @elgrillo83 can you check this? 
http://guides.rubyonrails.org/active_record_validations.html#custom-methods

When I modify the validation methods to add an error message into the record it should be fine. Are you with me?

**Comment by DenniJensen** [comment]
WIP

File: app/models/function.rb (hunk 3 above)
Comment (elgrillo83) [inline app/models/function.rb]: Yes, that's the one @DenniJensen, something like:
```ruby
validate :correct_time_order
...
private

def correct_time_order
  errors.add('message') unless correct?
end
```

**Comment by DenniJensen** [comment]
Testes with live database example.

File: app/views/functions/new.haml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Geht nicht du spacko
```
Comment (elgrillo83) [inline app/views/functions/new.haml]: ??

**Pull request merged**